check-license fails on .github/act/release-event.json (JSON has no comment syntax for a license header)

## Summary

The `check-license` job (apache/skywalking-eyes `license-eye`) fails on `.github/act/release-event.json`, added in #1259. JSON has no comment syntax, so `license-eye` cannot insert or match an Apache license header for it.

## Evidence

From the `check-license` job on #1259 ([run](https://github.com/oras-project/oras-go/actions/runs/30592122972/job/91148168089)):

```
INFO  Totally checked 246 files, valid: 216, invalid: 1, ignored: 29, fixed: 0
WARNING Failed to determine the comment style of file: .github/act/release-event.json
ERROR the following files don't have a valid license header:
.github/act/release-event.json
ERROR one or more files does not have a valid license header
##[error]Process completed with exit code 1.
```

It is the only offending file — the other three files in #1259 (`release.yml`, `.goreleaser.yaml`, `RELEASES.md`) carry the standard header and pass.

## Cause

A `.json` file cannot hold a comment, so there is no way to add the license header the checker requires. This is a checker-configuration gap, not a missing header.

## Suggested fix

Add the file (or the directory) to the ignore list in `.github/licenserc.yml`, alongside the 29 paths already ignored:

```yaml
header:
  paths-ignore:
    - '.github/act/release-event.json'
    # or, to cover future mock event payloads:
    # - '.github/act/**'
```

## Notes

- #1259 was merged with this check red; this issue tracks a follow-up to make `check-license` green again on `main`.
- If a `release/*` forward-port of this workflow lands on `v2`, the same ignore entry will be needed there.

--- a/.github/licenserc.yml
+++ b/.github/licenserc.yml
@@ -32,6 +32,7 @@ header:
     - '**/*.md'
     - 'CODEOWNERS'
     - 'LICENSE'
+    - '.github/act/release-event.json'
     - 'go.mod'
     - 'go.sum'
     - '**/testdata/**'

